interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,7 +35,373 @@ components:
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Unique serial number of the unmanned aircraft or, if the unmanned aircraft is privately built, the unique serial number of the add-on. This is expressed in the ANSI/CTA-2063 Physical Serial Number format. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 1.
+        operation_mode:   
+          $ref: '#/components/schemas/OperationMode'
+        operation_category:
+          type: string
+          enum: 
+            - Open
+            - Specific
+            - Certified
+          example: "Open"
+          description: |-
+            Category of UAS operation (‘open’, ‘specific’, ‘certified’) as defined in COMMISSION DELEGATED REGULATION (EU) 2019/945. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 4.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            description: Specify an identification technology that will be used for this flight
+            example: ['Broadcast','Network']
+        connectivity_methods:
+          type: array
+          items:
+            type: string
+          example: ['radio']
+          description: Connectivity methods. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 7.
+        endurance_minutes: 
+          type: number
+          description: Endurance of the UAS. This is expressed in minutes. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 8.
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: The URL at which the applicable emergency procedure in case of a loss of command and control link may be retrieved. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 9.
+        operator_id:
+          type: string
+          example: "CHE0n4fujx7nwanb-fwi"
+          description: ID assigned to the operator per the EN4709-02 standard
+      required:
+      - uas_serial_number
+      - operation_mode
+      - uas_class
+      - identification_technologies
+      - operator_id
+      - operation_category
+      - connectivity_methods
+      - endurance_minutes
+      - emergency_procedure_url
+
+    OperationMode:
+      type: string
+      enum:
+        - Undeclared
+        - Vlos
+        - Bvlos
+      description: Specify if the operation is a `vlos` or `bvlos` operation. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 2.
```

Comment:
nit: In plain English, VLOS and BVLOS should be all caps because it is an acronym (visual line of sight).  (How they are in the enum values [looks fine](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined) to me)

Reply:
I have made it all caps now. The `Undeclared` is kept as is.